Modules/Sources/PointOfSale/Presentation/Settings/PointOfSaleSettingsHardwareDetailView.swift

```diff
@@ -161,6 +168,65 @@ struct PointOfSaleSettingsHardwareDetailView: View {
         }
     }
 
+    private var cardReadersView: some View {
+        VStack(spacing: POSSpacing.none) {
+            POSPageHeaderView(
+                title: Localization.cardReadersTitle,
+                backButtonConfiguration: .init(state: .enabled, action: {
+                    navigationPath.removeLast()
+                }, buttonIcon: "chevron.left"))
+            .foregroundColor(.posSurface)
+
+            List {
+                if case .connected = posModel.cardReaderConnectionStatus {
+                    VStack(spacing: POSPadding.xSmall) {
+                        HStack {
+                            Text(Localization.readerModelTitle)
+                                .font(.posBodyMediumRegular())
+                                .foregroundStyle(.primary)
+                            Spacer()
+                            Text(cardReaderName)
+                                .font(.posBodyMediumRegular())
+                                .foregroundStyle(.secondary)
+                        }
+                        .padding()
+                        HStack {
+                            Text(Localization.readerBatteryTitle)
+                                .font(.posBodyMediumRegular())
+                                .foregroundStyle(.primary)
+                            Spacer()
+                            Text(formattedBatteryLevel)
```

Comment:
It's preexisting, and low priority, but the formatted battery level isn't localized properly at the moment, IMO. We should use a number formatter for this, so that the correct percentage symbol is used (e.g. in Arabic) and it goes in the correct place.